package.json

```diff
@@ -33,12 +33,15 @@
   "dependencies": {
     "@nestjs/common": "^9.3.12",
     "@nestjs/core": "^9.3.12",
+    "@nestjs/graphql": "^11.0.0",
```

Comment:
does it mean even if the main project using this library doesn't adopt the Graphql at all, the package `@nestjs/graphql` is still installed?

Reply:
yes, the interceptor depends on `@nestjs/graphql` to detect if a request is graphql.